### PR TITLE
Feature/endpoints frontend candidates

### PR DESCRIPTION
**Pull request by JonasVervloet**
/candidates endpoint returns a json list with name, familyName and listName

### CODE REVIEW

File: src/main.py [hunk 1]
```diff
@@ -6,6 +6,8 @@
 from rdflib import Graph, RDF
 from rdflib.namespace import FOAF
 from os import environ
+import requests
```
Comment (juliavdkris) [inline src/main.py:10]: Add `requests` to `requirements.txt`

File: src/main.py [hunk 2]
```diff
@@ -64,6 +66,38 @@ async def r_get(req, name):
     return response.json(valid_web_ids)
 
 
+@app.route('/candidates', methods=['GET'])
+async def get_handler(req):
+    sparql_db_url = environ.get('SPARQL_DBNAME')
```
Comment (juliavdkris) [inline src/main.py]: I'm being really pedantic here, but maybe rename the environment variable to `SPARQL_HOST`/`SPARQL_URL` or something like that, as it's the whole URL and not just the database name

**Review by juliavdkris — CHANGES_REQUESTED** [review]
Some small changes, but looks great! Please wait a bit before merging, I'm gonna try to fix the weird environment variable bug first (as it will probably also occur on the Swarm server)

File: src/main.py (hunk 1 above)
Comment (JonasVervloet) [reply src/main.py:10]: I'm on it!

File: src/main.py (hunk 2 above)
Comment (JonasVervloet) [reply src/main.py]: I'm on it!

**Review by juliavdkris — APPROVED** [review]

**Pull request merged**